package.json

```diff
@@ -8,21 +8,22 @@
   "license": "MIT",
   "private": false,
   "scripts": {
-    "build": "yarn --cwd client install && yarn --cwd client build",
     "start-server": "bin/rails server -e test -p 5017",
     "cy:run": "cypress run --project ./spec --config video=false",
     "cy:open": "cypress open --project ./spec",
     "cy:ci": "start-server-and-test start-server http://localhost:5017 cy:run",
     "deploy": "cp -a client/build/. public/",
-    "heroku-postbuild": "yarn build && yarn deploy",
-    "start": "yarn --cwd client start",
-    "test": "yarn --cwd client test",
+    "heroku-postbuild": "yarn install-all && yarn build && yarn deploy",
+    "build": "yarn --cwd client build",
     "lint": "yarn --cwd client lint",
     "lint:fix": "yarn --cwd client lint --fix",
-    "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md|vue)\"",
-    "install": "yarn --cwd client install",
+    "start": "yarn --cwd client start",
+    "test": "yarn --cwd client test",
+    "test-once": "yarn --cwd client test --watchAll=false",
     "ad": "yarn --cwd client add",
-    "rm": "yarn --cwd client remove"
+    "rm": "yarn --cwd client remove",
+    "install-all": "yarn install && yarn --cwd client install",
+    "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md|vue)\""
```

Comment:
`prettier` isn't available in the root `node_modules`. We might want to install it.